canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -35,6 +38,61 @@ $toolbar-divider-width: 1px;
 
 	/* Prevent elements from being dragged onto canvas */
 	-webkit-user-drag: none;
+	&.toolbar-size-small {
+		.toolbar-left-bar {
+			padding: 0;
+			.toolbar-popover-list {
```

Comment:
This change is also specific to table-buttons as well. We only want the menu of toolbars inside table-buttons to appear on the left. This should not affect any toolbar of size small

Reply:
moved into the `table-buttons.scss `
<img width="947" alt="image" src="https://user-images.githubusercontent.com/89609176/199601526-8db6a448-660e-4b9e-9a37-f30a01b2fbe6.png">